### PR TITLE
Improve clarity of log messages

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: apps/DisplayCluster/main.cpp [hunk 1]
```diff
@@ -52,45 +52,52 @@
...
     {
-        put_flog(LOG_FATAL, "Could not initialize application. %s", e.what());
+        put_flog( LOG_FATAL, "Could not initialize application. %s", e.what( ));
         return EXIT_FAILURE;
     }
 
     app->exec(); // enter Qt event loop
 
-    put_flog(LOG_INFO, "quitting");
+    put_flog( LOG_DEBUG, "waiting for threads to terminate..." );
```
Comment (tribal-tec) [inline apps/DisplayCluster/main.cpp]: ..to finish...

File: dc/core/ContentLoader.cpp [hunk 2]
```diff
@@ -53,9 +54,15 @@ bool ContentLoader::load( const QString& filename,
                           const QPointF& windowCenterPosition,
                           const QSizeF& windowSize )
 {
+    put_flog( LOG_INFO, "opening: '%s'", filename.toLocal8Bit().constData( ));
+
     ContentPtr content = ContentFactory::getContent( filename );
     if( !content )
+    {
+        put_flog( LOG_WARN, "ignoring unsupported file: '%s'",
```
Comment (tribal-tec) [inline dc/core/ContentLoader.cpp]: duplicate from dc/core/ContentFactory.cpp:136?

File: dc/core/DisplayGroup.cpp [hunk 3]
```diff
@@ -63,7 +63,7 @@ void DisplayGroup::addContentWindow( ContentWindowPtr contentWindow )
 {
     BOOST_FOREACH( ContentWindowPtr existingWindow, contentWindows_ )
     {
-        if ( contentWindow->getID() == existingWindow->getID( ))
+        if( contentWindow->getID() == existingWindow->getID( ))
         {
             put_flog( LOG_WARN, "A window with the same id already exists!" );
```
Comment (tribal-tec) [inline dc/core/DisplayGroup.cpp]: LOG_DEBUG?

File: dc/core/DisplayGroup.cpp [hunk 4]
```diff
@@ -179,6 +179,8 @@ DisplayGroup& DisplayGroup::operator=( const DisplayGroup& displayGroup )
 
 void DisplayGroup::clear()
 {
+    put_flog( LOG_INFO, "removing %i windows", contentWindows_.size( ));
```
Comment (tribal-tec) [inline dc/core/DisplayGroup.cpp]: LOG_DEBUG

File: dc/core/DynamicTexture.cpp [hunk 5]
```diff
@@ -337,9 +339,9 @@ void DynamicTexture::loadImage()
         }
     }
 
-    if(scaledImage_.isNull())
+    if( scaledImage_.isNull( ))
     {
-        put_flog(LOG_ERROR, "failed to load the image.");
+        put_flog( LOG_ERROR, "loading failed." );
```
Comment (tribal-tec) [inline dc/core/DynamicTexture.cpp]: put name/uri of image, otherwise not very helpful :)

File: dc/core/DynamicTexture.cpp [hunk 6]
```diff
@@ -568,19 +570,21 @@ QRectF DynamicTexture::getImageRegionInParentImage(const QRectF& imageRegion) co
...
 {
     // if we're in the starting node, we must ascend
-    if(start == this)
+    if( start == this )
     {
-        if(isRoot())
+        if( isRoot( ))
         {
-            put_flog(LOG_ERROR, "starting from root object and cannot ascend");
+            put_flog( LOG_ERROR, "root object has no parent!" );
```
Comment (tribal-tec) [inline dc/core/DynamicTexture.cpp]: dito. moreover, LOG_DEBUG?

File: apps/DisplayCluster/main.cpp (hunk 1 above)
Comment (rdumusc) [inline apps/DisplayCluster/main.cpp]: done

File: dc/core/ContentLoader.cpp (hunk 2 above)
Comment (rdumusc) [inline dc/core/ContentLoader.cpp]: Yes, this called for adaptations in the Factory code, which was incorrect. Done.

File: dc/core/DisplayGroup.cpp (hunk 3 above)
Comment (rdumusc) [inline dc/core/DisplayGroup.cpp]: OK, this never happens anyway ;-) Done

File: dc/core/DisplayGroup.cpp (hunk 4 above)
Comment (rdumusc) [inline dc/core/DisplayGroup.cpp]: My idea was to have this info in release mode as well, to see when the users clean the screen. I think it help understanding the sequence of events that happen.

File: dc/core/DynamicTexture.cpp (hunk 5 above)
Comment (rdumusc) [inline dc/core/DynamicTexture.cpp]: Done

File: dc/core/DynamicTexture.cpp (hunk 6 above)
Comment (rdumusc) [inline dc/core/DynamicTexture.cpp]: Done

**Comment by tribal-tec** [comment]
+1

**Pull request merged**